README.md

```diff
@@ -1,17 +1,16 @@
-# WinUI 3 Gallery 
+![WinUI Gallery Hero Image](assets/hero.png)
 
-[![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status%2Fmicrosoft.WinUI-Gallery?branchName=main)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=612&branchName=main) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/your/your-project/blob/master/LICENSE)
+<h1 align="center">
+    WinUI 3 Gallery
+</h1>
+<p align="center">
+  <a href="https://apps.microsoft.com/detail/9P3JFPWWDZRC?launch=true&mode=mini">
+    <img src="assets/storeBadge.png" width="200" /></a>
+</p>
 
-![WinUI Screenshot](README_Images/WinUIGalleryLanding.PNG)
+## ⭐️ Accompanying samples for [WinAppSDK](https://github.com/microsoft/WindowsAppSDK)

```

Comment:
I think "Accompanying Sample App for WinUI3" needs to be the second line so that there is no confusion between the different repos.